plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -369,6 +369,13 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
                 )
             }
 
+            is CodeModernizerStartJobResult.UploadArtifactToS3Failure -> {
+                CodeModernizerJobCompletedResult.UnableToCreateJob(
+                    message("codemodernizer.notification.warn.unable_to_start_job_due_to_upload_failure"),
```

Comment:
Needs to be:
message("codemodernizer.notification.warn.unable_to_start_job_due_to_upload_failure", URL)

to replace the `{0}`

Reply:
We dont need to pass here I think so as this will be retrieved directly from https://github.com/aws/aws-toolkit-jetbrains/pull/4339/files#diff-d5d5f0160522116b64d9ad48daa882f8f802d086797a0bfc86cc0363b1f43c3eR158

As I followed the above `CodeModernizerStartJobResult.UnableToStartJob `

Reply:
I don't think it's going to be retrieved, I've never used it like that